tests/test_meta/test_ordinal_classification.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pytest
+from sklearn.linear_model import LinearRegression, LogisticRegression, RidgeClassifier
+
+from sklego.common import flatten
+from sklego.meta import OrdinalClassifier
+from tests.conftest import classifier_checks, general_checks, select_tests
+
+
+@pytest.fixture
+def random_xy_ordinal():
+    np.random.seed(42)
+    X = np.random.normal(0, 2, (1000, 3))
+    y = np.select(condlist=[X[:, 0] < 2, X[:, 1] > 2], choicelist=[0, 2], default=1)
+    return X, y
+
+
+@pytest.mark.parametrize(
+    "test_fn",
+    select_tests(
+        flatten([general_checks, classifier_checks]),
+        exclude=[
+            "check_dict_unchanged",
+            "check_fit2d_1feature",
+            "check_classifier_data_not_an_array",
+            "check_classifiers_classes",
+            "check_classifiers_train",
```

Comment:
This is a fair amount of skipping, yet:
- coverage is still same
- the internal estimator will most likely deal directly with these cases (e.g. data not array)

Reply:
Yeah, seems fair.